src/components/Shared/LinePlot/PlotContainer/PlotContainerComponent.tsx

```diff
@@ -379,13 +379,21 @@ export class PlotContainerComponent extends React.Component<PlotContainerProps>
                     data: value,
                     fill: false,
                     lineTension: 0,
-                    borderColor: this.props.multiPlotBorderColor ? this.props.multiPlotBorderColor.get(key) : lineColor,
                     showLine: true,
                     steppedLine: this.props.interpolateLines ? false : "middle",
                     borderWidth: 1,
                     pointRadius: 0,
                     backgroundColor: this.props.dataBackgroundColor ? this.props.dataBackgroundColor : []
                 };
+                if (opacity < 1) {
+                    multiPlotDatasetConfig.borderColor = lineColor;
+                } else {
+                    if (this.props.multiPlotBorderColor) {
+                        multiPlotDatasetConfig.borderColor = this.props.multiPlotBorderColor.get(key);
```

Comment:
why does this not also happen if opacity = 1?

Reply:
just want to make lineColor stay blue like other plots, I will use "hexStringToRgba" for adding opacity.

Reply:
Shouldn’t the line’s RGB color be the same, no matter what the progress is, and only the opacity changed?

Reply:
will do